packages/story-editor/src/index.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import App from './editorApp';
+
+export * from './components/transform';
+export * from './app/config';
+export * from './components/previewPage';
+
+export { default as base64Encode } from './utils/base64Encode';
+export { default as getStoryPropsToSave } from './app/story/utils/getStoryPropsToSave';
+export { default as FontContext } from './app/font/context';
+export { default as useLoadFontFiles } from './app/font/actions/useLoadFontFiles';
+export { default as localStore } from './utils/localStore';
+export { default as StoryPropTypes } from './types';
+export { default as Modal } from 'react-modal';
```

Comment:
Should this be exported from 'design-system'? Design system also seems to import and use it as well..

Reply:
Good question.

Looks like the design system could just do

```
import Modal from 'react-modal';
const { setAppElement } = Modal;
export { setAppElement };
```

And then story-editor and wp-story-editor could import both from there

Reply:
Done in https://github.com/google/web-stories-wp/pull/8635